src/main/java/io/github/pylonmc/pylon/base/PylonBlocks.java

```diff
@@ -61,6 +64,17 @@ private PylonBlocks() {
     );
     static {
         GRINDSTONE.register();
+        Grindstone.Recipe.RECIPE_TYPE.addRecipe(new Grindstone.Recipe(
+            pylonKey("string_from_bamboo"),
+            new RecipeChoice.ExactChoice(new ItemStack(Material.BAMBOO)),
```

Comment:
```suggestion
            new RecipeChoice.MaterialChoice(Material.BAMBOO),
```
cleaner and does the same thing